components/AppButton.vue

```diff
@@ -1,12 +1,31 @@
 <template>
-  <button
-    class="shadow bg-teal-400 hover:bg-teal-400 focus:shadow-outline focus:outline-none text-white font-bold py-2 px-4 rounded"
-    :data-cy="data_cy"
-  >
+  <button class="appButtonStyle" :data-cy="data_cy">
     <slot></slot>
   </button>
 </template>
 
 <script>
 export default { name: 'AppButton', props: ['data_cy'] }
 </script>
+
+<style >
+.appButtonStyle {
+  @apply shadow;
```

Comment:
Good,  just a comment,  you can use a single line for all.

Reply:
i tried using a single line but it gave me red marks error but it seem to compile still with the red marks. But i ended up doing it individually .

Reply:
I know it is a eslint thing it is annoying but it compiles

Reply:
ah okay got it thanks Marco that makes things more easier now